### PR TITLE
Use external css style

### PR DESCRIPTION
**Pull request by VCneverdie**
- Add css class to each element
- Move all test related files to `/render/tests` for a clean architecture
- Fix readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +19,10 @@ import lightmd
 
 with open("your.md", "r") as md_file:
     ## Parse content
-    parsed_content = lightmd.parse_md_to_ast(md_file) 
+    parsed_content = lightmd.parse_md_to_ast(md_file.read()) 
     ## Render to HTML
     rendered_html = lightmd.render_html(parsed_content)
+    ## Render to HTML with custom css style
+    rendered_html = lightmd.render_html(parsed_content, "/path/to/css_style.css")
```
Comment (Czhang0727) [inline README.md:26]: try to avoid magic string

File: lightmd/render/html_render.py [hunk 2]
```diff
@@ -9,46 +10,96 @@ def get_html_format(node):
...
-    #print(HTML_FORMAT[HorizontalRule])
+
+def render(root, css_path=None):
+    # print(HTML_FORMAT[HorizontalRule])
+    global STYLE_MAP
+    STYLE_MAP = parse_css(css_path)
     return root.render(get_html_format)
+
+
+def parse_css(css_path):
```
Comment (andysim3d) [inline lightmd/render/html_render.py:77]: We don't require that method.

File: lightmd/render/tests/html_render_test.py [hunk 3]
```diff
@@ -130,18 +148,46 @@ def test_list_wrapper_render(content, ordered, html):
...
-    (r'<html> rand tag </html>',
-     r"<blockquote><html> rand tag </html></blockquote>", [TextBlock('<html> rand tag </html>')]),
-])
+
+@pytest.mark.parametrize("content, html, children",
+                         [(r'abc',
+                           r"<blockquote>abc</blockquote>",
+                           [TextBlock('abc')]),
+                             (r'<html> rand tag </html>',
+                              r"<blockquote><html> rand tag </html></blockquote>",
```
Comment (andysim3d) [inline lightmd/render/tests/html_render_test.py:157]: why `<blockquote><html>`?

**Review by andysim3d — CHANGES_REQUESTED** [review]

File: lightmd/render/html_render.py (hunk 2 above)
Comment (VCneverdie) [reply lightmd/render/html_render.py:77]: Based on our last discussion, I decide to stay with this approach. It allows users to specify multiple classes for one kind of element.

File: lightmd/render/tests/html_render_test.py (hunk 3 above)
Comment (VCneverdie) [reply lightmd/render/tests/html_render_test.py:157]: This change was caused by autopep8. I assume the original purpose is to test blockquote with multiple content.

**Review by andysim3d — APPROVED** [review]

**Pull request merged**